snake-game/snake.py

```diff
@@ -1,18 +1,23 @@
 import curses 
 from random import randint
 
+#constants
+
+width = 58
+height = 18 
```

Comment:
Use 60 and 20 here and adapt the other formulas accordingly. Also for constants use UPPERCASE